### PR TITLE
Add Nuspec config for packaging SDK

### PR DESCRIPTION
**Pull request by poornas**
- Add Minio.nuspec to project to package SDK for Nuget
- remove folder Minio.Api from source code as it is not being used.
- Add LICENSE to SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,4 +156,4 @@ namespace FileUploader
 
 ## Explore Further
 * [Complete Documentation](https://docs.minio.io)
-* [Minio .NET SDK API Reference](https://docs.minio.io/docs/.NET-client-api-reference)
\ No newline at end of file
+* [Minio .NET SDK API Reference](https://docs.minio.io/docs/.NET-client-api-reference)
```
Comment (harshavardhana) [inline README.md]: Are you sure this is going to be '.NET' saved as is on docs.minio.io why not keep it simple as 'dotnet-client-api-reference'

Comment (poornas) [inline README.md]: actually this was @deekoder's suggestion. dotnet- prefix definitely sounds better. will change it

Comment (harshavardhana) [inline README.md]: Keeping it as simple predictable URL makes it easy IMHO.

File: README.md [hunk 2]
```diff
@@ -21,7 +24,7 @@ To connect to an Amazon S3 compatible cloud storage service, you will need to sp
 | :---         |     :---     |
 | endpoint   | URL to object storage service.   | 
 | accessKeyID | Access key is the user ID that uniquely identifies your account. |
```
Comment (harshavardhana) [inline README.md]: Perhaps `accessKeyID` can be called as `accessKey`  just like `secretKey`

File: README.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 
-Minio Client SDK provides higher level APIs for Minio and Amazon S3 compatible cloud storage services. For a complete list of APIs and examples, please take a look at the [Dotnet Client API Reference](https://docs.minio.io/docs/dotnet-client-api-reference). This document assumes that you have a working VisualStudio development environment.  
+
+
+Minio Client SDK provides higher level APIs for Minio and Amazon S3 compatible cloud storage services.For a complete list of APIs and examples, please take a look at the [Dotnet Client API Reference](https://docs.minio.io/docs/dotnet-client-api-reference).This document assumes that you have a working VisualStudio development environment.  
 
 ## Minimum Requirements
-  .NET 4.5 or higher
-  Visual Studio 2015 
+  .NET 4.5.2, .NetCoreApp 1.0.0 or higher
```
Comment (harshavardhana) [inline README.md]: Isn't it odd to keep this in two new lines? rather than bullet points..

File: README.md (hunk 2 above)
Comment (harshavardhana) [inline README.md]: `secretAccessKey` was kept to be closer to how AWS S3 calls it. but it is simpler to call it the way you changed it. We can change this across all our libraries as well.

File: Minio.nuspec [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        <dependency id="System.Xml.Linq" version="3.5.21022.801" />
+      </group>
+      <group targetFramework="netcoreapp1.0">
+        <dependency id="RestSharp.NetCore" version="105.2.3" />
+        <dependency id="Newtonsoft.Json" version="10.0.2" />
+        <dependency id="System.Reactive.Linq" version="3.1.1" />
+        <dependency id="System.Runtime.Serialization.Xml" version="4.3.0" />
+        <dependency id="Microsoft.Win32.Registry" version="4.3.0" />
+        <dependency id="Microsoft.Build.Utilities.Core" version="15.1.548" />
+
```
Comment (harshavardhana) [inline Minio.nuspec]: Is this space needed?

File: Minio.nuspec [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      </group>
+
```
Comment (harshavardhana) [inline Minio.nuspec]: Is this space needed?

File: Minio.nuspec [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    </dependencies>
+
```
Comment (harshavardhana) [inline Minio.nuspec]: Is this space needed?

**Comment by harshavardhana** [comment]
Looks like the nupkg generated is proper and can be published.. 

```
$ mv Minio.1.0.0.nupkg Minio.1.0.0.zip
$ unzip Minio.1.0.0.zip
Archive:  Minio.1.0.0.zip
 extracting: _rels/.rels             
  inflating: Minio.nuspec            
  inflating: lib/net452/Minio.dll    
  inflating: lib/netcore/Minio.dll   
 extracting: package/services/metadata/core-properties/a1ef63192b5545b4bf00cd7a578fa03a.psmdcp  
 extracting: [Content_Types].xml     
$ file lib/net452/Minio.dll
lib/net452/Minio.dll: PE32 executable (DLL) (console) Intel 80386 Mono/.Net assembly, for MS Windows
$ file lib/netcore/Minio.dll
lib/netcore/Minio.dll: PE32 executable (DLL) (console) Intel 80386 Mono/.Net assembly, for MS Windows
```

**Pull request merged**